pyaptamer/utils/__init__.py

```diff
@@ -1,19 +1,10 @@
 """Utils for the pyaptamer package."""
 
+__author__ = ["nennomp"]
 __all__ = [
-    "dna2rna",
```

Comment:
what is going on here? I think we should leave the exports as they are.

Reply:
replied above.

Reply:
where?